Plugins/Animation/CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ set(Qt5_LIBRARIES ${Qt5Core_LIBRARIES} ${Qt5Widgets_LIBRARIES} ${Qt5OpenGL_LIBRA
 target_link_libraries( Animation
                        ${Qt5_LIBRARIES}
                        ${RADIUM_LIBRARIES}
+					   ${GLBINDING_LIBRARIES}           # Radium dep
```

Comment:
Because I prefer tabs :-p
Sirously, this comes when switching between different configurations and indentation recommandation on several tools on several platforms ... I'll try to fix this for my next PR